patches/arcade/0010-Update-newtonsoft-json-version-for-arcade.patch

```diff
@@ -0,0 +1,23 @@
+From 804579b9f02916cd38d1e61952fbe852155caa5c Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Mon, 4 May 2020 15:09:20 +0000
+Subject: [PATCH] Update newtonsoft json version for arcade
```

Comment:
Purpose/issue?

Reply:
I added info to the patch.  There is an issue in arcade to get off of newtonsoft json.  I linked that.